zk_toolbox/crates/zk_inception/src/commands/prover/init.rs

```diff
@@ -0,0 +1,179 @@
+use common::{check_prover_prequisites, cmd::Cmd, logger, spinner::Spinner};
+use config::EcosystemConfig;
+use xshell::{cmd, Shell};
+use zksync_config::{
+    configs::{object_store::ObjectStoreMode, GeneralConfig},
+    ObjectStoreConfig,
+};
+
+use super::{
+    args::init::{ProofStorageConfig, ProofStorageGCSCreateBucket, ProverInitArgs},
+    utils::get_link_to_prover,
+};
+use crate::messages::{
+    MSG_CHAIN_NOT_FOUND_ERR, MSG_DOWNLOADING_SETUP_KEY_SPINNER, MSG_GENERAL_CONFIG_NOT_FOUND_ERR,
+    MSG_GETTING_GCP_PROJECTS_SPINNER, MSG_PROOF_COMPRESSOR_CONFIG_NOT_FOUND_ERR,
+    MSG_PROVER_CONFIG_NOT_FOUND_ERR, MSG_PROVER_INITIALIZED,
+};
+
+const PROVER_STORE_MAX_RETRIES: u16 = 10;
+
+pub(crate) async fn run(args: ProverInitArgs, shell: &Shell) -> anyhow::Result<()> {
+    check_prover_prequisites(shell);
+    let ecosystem_config = EcosystemConfig::from_file(shell)?;
+    let chain_config = ecosystem_config
+        .load_chain(Some(ecosystem_config.default_chain.clone()))
+        .expect(MSG_CHAIN_NOT_FOUND_ERR);
+    let mut general_config = chain_config
+        .get_zksync_general_config()
+        .expect(MSG_GENERAL_CONFIG_NOT_FOUND_ERR);
+
+    let spinner = Spinner::new(MSG_GETTING_GCP_PROJECTS_SPINNER);
+    let project_ids = get_project_ids(shell)?;
```

Comment:
there is an option to setup local based artifacts. 
should we request project ids in this case?